.drone.yml

```diff
@@ -0,0 +1,36 @@
+---
+kind: pipeline
+type: docker
+name: lint
+
+trigger:
+  event:
+    include:
+      - pull_request
+      - tag
+
+steps:
+  - name: build
+    image: node:12.14.1
+    commands:
+      - yarn --version
+      - yarn install
+      - yarn lint
+
+---
+kind: pipeline
+type: docker
+name: tests
+
+trigger:
+  event:
+    include:
+      - pull_request
+      - tag
+
+steps:
+  - name: unit test
+    image: node:12.14.1
```

Comment:
Lets use more recent version of node here. current lts is 14

Reply:
fixed